tasks/bump.js

```diff
@@ -77,8 +82,7 @@ module.exports = function(grunt) {
 
     var globalVersion; // when bumping multiple files
     var gitVersion;    // when bumping using `git describe`
-    var VERSION_REGEXP = /([\'|\"]?version[\'|\"]?[ ]*:[ ]*[\'|\"]?)([\d||A-a|.|-]*)([\'|\"]?)/i;
-
+    var VERSION_REGEXP = /([\'|\"]?version[\'|\"]?[ ]*:[ ]*[\'|\"]?)(\d+\.\d+\.\d+)(-rc\.?\d+)?([\w-]*)?([\'|\"]?)/i;
```

Comment:
Good point.  I've fixed the regex so that it will recognize the semverversion, but not the git portion of a 'git version'